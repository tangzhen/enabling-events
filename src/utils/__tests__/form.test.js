import {
  getCategoryOptions,
  getOrganizerOptions,
  getTypeOptions,
} from "../form";

describe("Form related module", () => {
  it("should return the organizer options", () => {
    const result = getOrganizerOptions();

    expect(result).toMatchSnapshot();
  });

  it("should return the type options", () => {
    const result = getTypeOptions();

    expect(result).toMatchSnapshot();
  });

  it("should return the category options", () => {
    const result = getCategoryOptions();

    expect(result).toMatchSnapshot();
  });
});
