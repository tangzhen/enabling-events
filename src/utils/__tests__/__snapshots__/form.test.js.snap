// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form related module should return the category options 1`] = `
Array [
  Object {
    "label": "A",
    "value": 1,
  },
  Object {
    "label": "B",
    "value": 2,
  },
  Object {
    "label": "C",
    "value": 3,
  },
]
`;

exports[`Form related module should return the organizer options 1`] = `
Array [
  Object {
    "label": "A",
    "value": 1,
  },
  Object {
    "label": "B",
    "value": 2,
  },
  Object {
    "label": "C",
    "value": 3,
  },
]
`;

exports[`Form related module should return the type options 1`] = `
Array [
  Object {
    "label": "A",
    "value": 1,
  },
  Object {
    "label": "B",
    "value": 2,
  },
  Object {
    "label": "C",
    "value": 3,
  },
]
`;
